Header header

uint32 DISCONNECTED=0
uint32 CONNECTED=1

uint32 ins_stream_type
uint32 rtk_stream_in_type
uint32 rkt_stream_out_type
